src/scripts/main.js

```diff
@@ -1,3 +1,53 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const body = document.querySelector('body');
+
+const promise1 = new Promise((resolve, reject) => {
+  logo.onclick = () => {
+    resolve();
+  };
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(reject, 3000);
+});
+
+promise1.then(
+  function (value) {
+    myDisplayer(value);
+  },
+  function (error) {
+    myDisplayer2(error);
+  },
+);
+
+promise2.then(
+  function (value) {
+    myDisplayer(value);
+  },
+  function (error) {
+    myDisplayer2(error);
+  },
```

Comment:
Similar to the previous comment, the `then` method should only have two arguments. The third argument here is unnecessary and should be removed.